corehq/messaging/smsbackends/amazon_pinpoint/forms.py

```diff
@@ -0,0 +1,29 @@
+from corehq.apps.sms.forms import BackendForm
+from dimagi.utils.django.fields import TrimmedCharField
+from crispy_forms import layout as crispy
+from django.utils.translation import ugettext_lazy as _
+
+
+class PinpointBackendForm(BackendForm):
+    project_id = TrimmedCharField(
+        label=_("Project ID"),
```

Comment:
Are these fields required? If yes, I think they should specify `required=True`

Reply:
TrimmedCharField is a subclass of Field class. Here is the class constructor :
![image](https://user-images.githubusercontent.com/16632752/83839515-887bb700-a6fc-11ea-97f7-4bbec041f8d8.png)

So required field is True by default. This is counterintuitive and in order to avoid confusion, I'll just explicitly specify required=True. I fixed that for Infobip backend as well.